### ISSUE TITLE
Move table classes under the classes/table folder

### ISSUE THREAD

**Issue by keevan** [catalyst/moodle-tool_dataflows#240]
I've seen other table classes organised this way and it seems fitting.

**Comment by brendanheywood** [comment]
I don't think table is actually a valid namepsace? I think it *should* be, but it isn't

**Comment by keevan** [comment]
Yeah so I'm not too sure about this one. Here's a few cases that have some form of `classes/table(s)/some_class_for_with_table.php`

reportbuilder
atto
report/infectedfiles
question/bank/usage
enrol/lti

![image](https://user-images.githubusercontent.com/9924643/175860463-2d7333b5-50fa-4575-820f-8736601441b1.png)

I'm fine to leave it where it is for now and we can make a decision about where they should be organised later.

**Comment by keevan** [comment]
Circling back to this, I think it's better to have it in `classes/local/table/` if not `classes/table` purely for file organisation. 

Not priority, but would be something nice to fix along the way if we ever add another table class.
